go/vt/binlog/binlog_streamer.go

```diff
@@ -52,9 +52,20 @@ var (
 	}
 )
 
+// FullBinlogStatement has all the information we can gather for an event.
```

Comment:
We agreed offline that is not needed right now, because we won't send this over the network.